var/spack/repos/builtin/packages/hip-tensor/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class HipTensor(CMakePackage, ROCmPackage):
+    """AMD’s C++ library for accelerating tensor primitives"""
+
+    homepage = "https://github.com/ROCmSoftwarePlatform/hipTensor"
+    git = "https://github.com/ROCmSoftwarePlatform/hipTensor.git"
+    url = "https://github.com/ROCmSoftwarePlatform/hipTensor/archive/refs/tags/rocm-5.7.1.tar.gz"
+    tags = ["rocm"]
```

Comment:
can you update the homepage and git and url to https://github.com/ROCm/hipTensor ; https://github.com/ROCm/hipTensor.git ; https://github.com/ROCm/hipTensor/archive/refs/tags/rocm-5.7.0.tar.gz